src/plugins/orangehrmClaimPlugin/entity/Decorator/ClaimRequestDecorator.php

```diff
@@ -99,4 +104,31 @@ public function setEmployeeByEmpNumber(int $empNumber): void
         $employee = $this->getReference(Employee::class, $empNumber);
         $this->getClaimRequest()->setEmployee($employee);
     }
+
+    /**
+     * @return float
+     */
+    public function getTotalExpense(): float
```

Comment:
is there any other way to improve this. since if you have 50 items in API response then, this will call 50 times